ballerina-shell/modules/shell-core/src/main/java/io/ballerina/shell/ModuleImporter.java

```diff
@@ -0,0 +1,156 @@
+/*
```

Comment:
This class should move to utils.

Reply:
Updated in 7206767a98cd266b722f016fca60ecf1fac9aec7